mozdownload/scraper.py

```diff
@@ -242,6 +243,51 @@ def detect_platform(self):
         else:
             return "%s%d" % (mozinfo.os, mozinfo.bits)
 
+    def get_checksum_url(self):
+        """Get the checksum url for current version."""
+        filename = 'SHA512SUMS'
+        url = os.path.split(os.path.split(os.path.split(self.url)[0])[0])[0] + '/' + filename
+        return url
```

Comment:
Ah, I see. So yes, it's the correct one. But the path or regex of this file has to be a class variable, so that it can be adjusted for each individual scraper class, and the base scraper has the correct one available.